extension/fabric/FabricConnection.ts

```diff
@@ -136,6 +136,44 @@ export abstract class FabricConnection {
         }
     }
 
+    public async getChannelPeers(channelName: string, peerNames: string[]): Promise<Client.ChannelPeer[]> {
```

Comment:
this function doesn't need to be public you only use it in the child class you can just make it protected